.shellcheckrc

```diff
@@ -0,0 +1,2 @@
+# Disable file checks (otherwise every use of `gha-tools` will get flagged)
+disable=SC1091
```

Comment:
I think this file should be added to the `ci-codeowners` list:

https://github.com/rapidsai/rapidsmpf/blob/0dc1d66b9005ce6db6c7c0cb0d2cb3bf0eded651/.github/CODEOWNERS#L12-L14

Reply:
Ahh, good call.  I'll do that